### PR TITLE
Change Readme

### PR DESCRIPTION
**Pull request by Spidy123222**
This changes the readme to one meant for Sidestore. This is currently a draft since the project is still progressing and you guys can edit this if you feel it needs to be changed before merging.

### CODE REVIEW

**Comment by Simonx22** [comment]
Is this project called SideStore or Sidestore? The GitHub organization and repo is called “SideStore” but in the readme it’s called “Sidestore”.

**Comment by Spidy123222** [comment]
> I was just about to comment the same thing, I’ve pulled out my iPad to point out some other things.

yeah wasn't quite sure if it should so I used Sidestore.

**Comment by jkcoxson** [comment]
It should be upper camel case (SideStore)

**Comment by Spidy123222** [comment]
> It should be upper camel case (SideStore)

It has been fixed now.

File: README.md [hunk 1]
```diff
@@ -1,50 +1,31 @@
...
-
-### AltKit
-AltKit is a shared framework that includes common code between AltStore and AltServer.
-
-### AltSign
-AltSign is my internal framework used by both AltStore and AltServer to communicate with Apple's servers and resign apps. For more info, check the [AltSign repo](https://github.com/rileytestut/altsign).
+### SideStore
+SideStore is a alternative to Altstore and is a sandboxed iOS application like Altstore. The SideStore app target contains the vast majority of AltStore's functionality, including all the logic for downloading and updating apps through SideStore.
 
-### Roxas
```
Comment (jkcoxson) [inline README.md:46]: We can leave Roxas, just change it to mention that it's Riley's framework. We're still using it as a dependency.

File: README.md [hunk 2]
```diff
@@ -1,50 +1,31 @@
...
-Roxas is my internal framework used across all my iOS projects, developed to simplify a variety of common tasks used in iOS development. For more info, check the [Roxas repo](https://github.com/rileytestut/roxas).
+### Netmuxd
+Netmuxd is a program that replaces Usbmuxd to be able to connect over a VPN reliably. It is coded in rust language and is a fully open source program that you can use.
```
Comment (jkcoxson) [inline README.md:28]: Maybe change to 'It is programmed in the Rust programming language and it open source'
And you can set Netmuxd to be a hyper link like [Netmuxd](https://github.com/jkcoxson/netmuxd)

File: README.md [hunk 3]
```diff
@@ -59,17 +40,37 @@ AltStore and AltServer are both fairly straightforward to compile and run if you
...
-* Twitter: [@rileytestut](https://twitter.com/rileytestut)
+AltWidget/Info.plist
+Change ALTAppGroups to your group
+
+AltStore/Info.plist
+Change ALTAppGroups to your app group name group.com.[Rick].SideStore
+Change ALTDeviceID to your device's UDID. You can fetch it with libimobiledevice if needed.
+
+AltStore/project.pbxproj
+Change all the DEVELOPMENT_TEAMs to your dev team ID. It's what AltStore appends to your bundle IDs.
```
Comment (jkcoxson) [inline README.md:66]: This is unnecessary because you said to do it in step 3

File: README.md (hunk 1 above)
Comment (Spidy123222) [reply README.md:46]: solved on latest commit

File: README.md [hunk 4]
```diff
@@ -59,17 +40,37 @@ AltStore and AltServer are both fairly straightforward to compile and run if you
...
+
+Shared/Extensions/Bundle+AltStore.swift
+Change the string "group.com.rileytestut.AltStore" to your group you are using.
+
+Build + run app! 🎉
+
+## Licensing
 
-Questions about AltStore in general? Make sure to read the FAQ at https://altstore.io/faq/
+Due to the licensing of some dependencies used by SideStore, I have no choice but to distribute SideStore under the **AGPLv3 license** because this is a fork of Altstore. That being said, our goal for SideStore is for it to be an open source project *anyone* can use without restrictions, so we explicitly give permission for anyone to use, modify, and distribute all *our* original code for this project in any form, with or without attribution, without fear of legal consequences (dependencies remain under their original licenses, however).
```
Comment (jkcoxson) [inline README.md:79]: Maybe just state that it's licensed under AGPLv3. I don't like the tone of this.

File: README.md (hunk 3 above)
Comment (Spidy123222) [reply README.md]: Changed in latest commit

File: README.md (hunk 4 above)
Comment (Spidy123222) [reply README.md]: Changed in latest commit

File: README.md (hunk 2 above)
Comment (Spidy123222) [reply README.md]: Changed in latest commit

File: README.md (hunk 1 above)
Comment (Spidy123222) [reply README.md:46]: i have added roxas thanks for mentioning its Changed in latest commit

File: README.md [hunk 5]
```diff
@@ -1,75 +1,77 @@
...
 
-### AltSign
-AltSign is my internal framework used by both AltStore and AltServer to communicate with Apple's servers and resign apps. For more info, check the [AltSign repo](https://github.com/rileytestut/altsign).
+### Netmuxd
+Netmuxd is a program that replaces Usbmuxd to be able to connect over a VPN reliably.  It is programmed in the Rust programming language and it is open source. You can set Netmuxd to be a hyper link like Netmuxd: [Netmuxd](https://github.com/jkcoxson/netmuxd)
 
 ### Roxas
-Roxas is my internal framework used across all my iOS projects, developed to simplify a variety of common tasks used in iOS development. For more info, check the [Roxas repo](https://github.com/rileytestut/roxas).
+Roxas is Riley Testuts internal framework from Altstore used across many of their iOS projects, developed to simplify a variety of common tasks used in iOS development. For more info, check the [Roxas repo](https://github.com/rileytestut/roxas).
```
Comment (jkcoxson) [inline README.md:31]: Testut's, Alt**S**tore

File: README.md [hunk 6]
```diff
@@ -1,75 +1,77 @@
...
-5. **(AltStore only)** Change the value for `ALTServerID` in the Info.plist to your AltServer's serverID. This is embedded by AltServer during installation to help AltStore distinguish between multiple AltServers on the same network, and you can find this by using a Bonjour browsing application and noting the serverID advertised by AltServer. This isn't strictly necessary, because if AltStore can't find the AltServer with the embedded serverID it still falls back to trying another AltServer. However, this will help in cases where there are multiple AltServers running (plus the error messages are more helpful).
-6. Build + run app! 🎉
 
-## Licensing
+5. **(SideStore app only)** Change the value for `ALTDeviceID` in the Info.plist to your device's UDID. Normally, SideStore embeds the device's UDID in AltStore's Info.plist during installation. When running through Visual Studio you'll need to set the value yourself or else Sidetore won't resign (or even install) apps for the proper device. You can achieve this by changing a few things to be able to build and use SideStore.
```
Comment (jkcoxson) [inline README.md:47]: Misspelled SideStore

Comment (Spidy123222) [reply README.md:47]: fixed it in https://github.com/SideStore/SideStore/pull/11/commits/be79f1a1f5d29ac16bd4a0ce4bb0e9aac85afd63

Comment (Spidy123222) [reply README.md]: This should fix that https://github.com/SideStore/SideStore/pull/11/commits/40660e459e0d662eca5fb6ffcd284fa2e9338994

File: README.md (hunk 5 above)
Comment (Spidy123222) [reply README.md]: fixed in https://github.com/SideStore/SideStore/pull/11/commits/40660e459e0d662eca5fb6ffcd284fa2e9338994

**Comment by jkcoxson** [comment]
I’ve already reviewed it smh, we’re waiting on 1 other approval

**Comment by Spidy123222** [comment]
> I’ve already reviewed it smh, we’re waiting on 1 other approval

Yeah sorry I miss clicked I didn’t know there was a way to request for a re-review at bottom of page sorry about that. I thought it was a way to refresh.

**Review by JoeMatt — APPROVED** [review]

**Pull request merged**